Automattic/woocommerce-payments
# Special tooltip content when pending balance is negative

Fixes https://github.com/Automattic/woocommerce-payments/issues/6051

#### Changes proposed in this Pull Request

- Show different tooltip content when pending balance is negative. I use the same tooltip content from available balance's tooltip when available balance is negative.
- Add a unit test and along with that, remove the use of `documentationUrls` in the unit test. This is so when there is a change in `documentationUrls` strings, it's detected in the unit test.

#### Testing instructions

* Make a large purchase. Can be cheap product but in large quantity.
* Refund the order.
* Notice the pending balance is negative.
* Click on the tooltip with [a link to the docs explaining about negative balance](https://woocommerce.com/document/woocommerce-payments/fees-and-debits/account-showing-negative-balance).

![Screenshot 2023-05-11 at 02 41 42](https://github.com/Automattic/woocommerce-payments/assets/73803630/fdc4db54-3eaf-4542-ab7c-1421584b5b71)

* With base branch, the tooltip stays the same even though pending balance is negative.

![Screenshot 2023-05-11 at 02 43 57](https://github.com/Automattic/woocommerce-payments/assets/73803630/93222edf-d60c-4f02-9bc3-c2315bbeeb62)

-------------------


**Post merge**
